### PR TITLE
File name convention 

### PR DESCRIPTION
**Pull request by ubergesundheit**
This is the first bunch of file/folder renames related to giantswarm/giantswarm#7915 according to the rules layed out in fmt.

This should be squashed I think

### CODE REVIEW

**Comment by oponder** [comment]
> This should be squashed I think

It will be squashed on merge, as configured in github.

File: src/components/app_catalog/AppList/AppList.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const AppList = ({ catalog, ...props }) => {
       }}
     >
       <DocumentTitle title={'Apps | Giant Swarm '}>
-        <>
+        <React.Fragment>
```
Comment (oponder) [inline src/components/app_catalog/AppList/AppList.js]: I guess things got a bit confused here, sorry for doing large changes while you're also doing the filename change.

Comment (ubergesundheit) [reply src/components/app_catalog/AppList/AppList.js]: I'm going to change this one to the short form in here

**Comment by ubergesundheit** [comment]
I'll try to fix the conflicts now and start another round of renamings ;)

File: src/components/app_catalog/AppList/AppList.js (hunk 1 above)
Comment (ubergesundheit) [reply src/components/app_catalog/AppList/AppList.js]: But I guess the Fragment isn't required at all.. Will remove it

**Pull request merged**